app/controllers/profiles_controller.rb

```diff
@@ -199,12 +200,11 @@ def profiles_for_category
                                   .translated_in_current_language_and_not_translated(I18n.locale)
     tag_names = @tags_in_category_published.pluck(:name)
     @tags_most_used_200_in_category = @tags_in_category_published.most_used(200)
-    @profiles = profiles_for_tag(tag_names)
+    @profiles = @profiles_for_pagination = profiles_for_tag(tag_names)
```

Comment:
I don't understand that code. :(

Reply:
I had to remove the `.includes()` part in `profiles_for_search`, in order to control it more finely in the `index` method for the `elsif params[:search]` case. You can see there that the `.includes(:translations, :image_attachment)` is now only applied for the value of `@profiles`, while everything else uses the value of `@profiles_for_pagination`. 
Since this separation to two different variables only applies to the `elsif params[:search]` section, in the other sections (like here) I had to only make sure that the variable `@profiles_for_pagination` exists and has the same value as `@profiles`.
This line says: take the value of `profiles_for_tag(tag_name)` and store it in both variables `@profiles` and `@profiles_for_pagination`.

I know this is a little bit complicated, but it's the only workaround I could find to cope with the way that the rails elasticsearch gem works with `.includes()`.

Reply:
Thank you! It is hard to understand!